association_backend/association/api/builder.py

```diff
@@ -0,0 +1,45 @@
+from dataclasses import field, make_dataclass
+from typing import List
```

Comment:
you can remove this file now and use

```
from pythonit_toolkit.api.builder import create_root_type
```

install pythonit-toolkit latest

Reply:
ok done